app/assets/javascripts/components/GlobalNav.js

```diff
@@ -32,13 +32,177 @@ define(['jquery', 'DoughBaseComponent', 'mediaQueries', 'utilities'], function($
     this._setUpMobileInteraction();
     this._setUpDesktopInteraction();
     this._setUpMobileAnimation();
+    this._setUpKeyboardEvents();
 
     $(window).on('resize', utilities.debounce($.proxy(this._setUpMobileAnimation, this), 100));
 
     return this;
   };
 
   /**
+   * Set up keyboard events
+   * Adds enhanced keyboard interation to nav
+   */
+  GlobalNav.prototype._setUpKeyboardEvents = function() {
+    var self = this;
+
+    this.$globalNav.keydown(function(e) {
+      switch (e.which) {
+        // down arrow
+        case 40:
+          e.preventDefault();
+          break;
+        // spacebar
+        case 32:
+          e.preventDefault();
+          break;
+      }
+    })
```

Comment:
Minor, but there's inconsistent semi-coloning throughout, if you run JSHint locally it should point them out.